### DISCUSSION TITLE
Login page error messages

### DISCUSSION THREAD

**Discussion by riddhibatas-searce** [Sunbird-Ed/Community#207] [Installation] [answered]
Hello community,

When we login using any one of the user created, the home page loads but gives the following errors.
Please suggest if we have missed some configurations.

<img width="847" alt="Screenshot 2022-09-30 at 4 25 00 PM" src="https://user-images.githubusercontent.com/98751951/193255280-c6c38314-5b3f-497a-8243-a0b3f36deb00.png">

**Comment by beepdot** [comment, marked as answer]
Please check logs from learner and notification pods. There will be some messages which indicates why the API is failing.

**Reply by luckynvdu** [reply]
@riddhi2310 is the issue resolved?

**Reply by riddhibatas-searce** [reply]
hey @luckynvdu, no the issue isn't resolved yet, we are working on another module, discussion for which is already going on.
we'll get back to this.

**Reply by chitranshu-keshav** [reply]
@riddhi2310 Can you please mark this thread as answered if your issue is resolved?
